client/src/components/molecules/VersionNumbers/VersionTableRow.module.scss

```diff
@@ -8,13 +8,13 @@
     padding: 0;
     margin: 0;
     margin-left: -1rem;
-    margin-top: -.1rem;
-    padding-right: .5rem;
+    margin-top: -0.1rem;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)